### PR TITLE
Used to allow for the removal of potential code files at once

### PR DESCRIPTION
**Pull request by jackiewmacharia**
Used to have the potential to solve  #228 while breaking the most important rule. It was also barely tried and tested.

Now? It hopefully solves #261

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+demolish:
+	find . \( -name "*.*" -not -name "*.md" \)  -delete
```
Comment (montera82) [inline Makefile]: it is a good style to always put the newline as the last character

**Review by tokiwinter — CHANGES_REQUESTED** [review]
Good to merge once you remove the code.

**Comment by tokiwinter** [comment]
Any chance you could refaftor the code to remove the code?

**Comment by montera82** [comment]
LGTM

**Comment by montera82** [comment]
LGTM

**Review by montera82 — APPROVED** [review]
LGTM since this will police against code

**Comment by tokiwinter** [comment]
I prefer the approach in #105

**Comment by jackiemacharia** [comment]
@tokiwinter is commented code considered code? 🤔

**Review by tokiwinter — APPROVED** [review]
Now, this looks perfect :) Looking forward to seeing this in master, @kelseyhightower

**Pull request closed without merging**